packages/server/validators/events.js

```diff
@@ -0,0 +1,229 @@
+const { param, body, check } = require("express-validator");
+
+const EventController = require("../controllers/events");
+const GuildController = require("../controllers/guilds");
+
+// validators are defined as an array of checks to perform with express-validator
+const createEventValidator = [
+  // example check for making sure guildId route parameter is a UUID
+  // modify this param("guildId") check for the other checks I listed in the PR review for "guildId"
+  param("guildId", "Invalid guild ID")
+    .trim()
+    .escape()
+    .exists({ checkFalsy: true })
+    .withMessage("guildId cannot be null or empty")
+    .isUUID()
+    .withMessage("Not a UUID")
+    .custom(async (value) => {
+      try {
+        await GuildController.getGuild(value); //Check if guild exists by getting it
+      } catch (error) {
+        //If event does not exists, then a NotFoundError is thrown from controller
+        throw new Error(error);
```

Comment:
Instead of throwing the Prisma error, throw a new error with a generic but descriptive message of what went wrong. We don't want to throw the Prisma error because it can contain sensitive info. Something like
```js
throw new Error(`No guild exists with an ID of ${value}`
```
would be good. This goes for the other custom validators that involve Prisma queries as well.